ItemGenerator/WordGenerator.cs

```diff
@@ -23,12 +22,10 @@ public string Generate(WordSequenceItem item, int seed)
                 if (item.Words == null || !item.Words.Override)
                     wordList.AddRange(_globalConfiguration.DefaultWords);
 
-                targetWord = wordList[randomSeed.Next(wordList.Count)];
+                targetWord = wordList[cryptoRandom(0, wordList.Count-1)];

```

Comment:
I fixed this issue. It still compiled and ran with a full rebuild, even before I fixed it...I don't know why.